src/rust/engine/src/cffi_build.rs

```diff
@@ -18,10 +18,32 @@ native engine binary, allowing us to address it both as an importable python mod
 
 */
 
+use std::fs;
+use std::io::{Read, Result};
+use std::path::{Path, PathBuf};
+
 fn main() {
-  gcc::Config::new()
-    // N.B. The filename of this source code - at generation time - must line up 1:1 with the
-    // python import name, as python keys the initialization function name off of the import name.
-    .file("src/cffi/native_engine.c")
-    .compile("libnative_engine_ffi.a");
+  let mut config = gcc::Config::new();
+
+  let path = PathBuf::from("src/cffi/native_engine.c");
+
+  // N.B. The filename of this source code - at generation time - must line up 1:1 with the
+  // python import name, as python keys the initialization function name off of the import name.
```

Comment:
this comment should probably move above the `let path = PathBuf::from("src/cffi/native_engine.c");` line, since it's talking about the `native_engine.c` bit.

Reply:
Done